<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>208</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>593</width>
    <height>208</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="inputDateLayout">
         <item>
          <widget class="QLabel" name="date1_label">
           <property name="text">
            <string>Date1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="first_Date"/>
         </item>
         <item>
          <widget class="QLabel" name="date2_label">
           <property name="text">
            <string>Date2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="second_Date"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="inputTimeLayout">
         <item>
          <widget class="QLabel" name="time1_label">
           <property name="text">
            <string>Time1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="first_Time"/>
         </item>
         <item>
          <widget class="QLabel" name="time2_label">
           <property name="text">
            <string>Time2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="second_Time"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="inputDeltalLayout">
         <item>
          <widget class="QLabel" name="delta_time_label">
           <property name="text">
            <string>DeltaTime</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTimeEdit" name="delta_Time"/>
         </item>
         <item>
          <widget class="QLabel" name="delta_date_day_label">
           <property name="text">
            <string>DeltaDateDay</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="delta_Date_Day"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="buttonDt1_minus_Dt2">
           <property name="text">
            <string>Date1 - Date 2</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="buttonT1_minus_T2">
           <property name="text">
            <string>Time1 - Time2</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="buttonDt2_minus_Dt1">
           <property name="text">
            <string>Date2 - Date1</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="buttonT2_minus_T1">
           <property name="text">
            <string>Time2 - Time1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="buttonT1_minus_DeltaTime">
           <property name="text">
            <string>Time1 - DeltaTime</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="buttonD1_minus_DeltaDateDay">
           <property name="text">
            <string>Date1 - DeltaDateDay</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="buttonT2_minus_DeltaTime">
           <property name="text">
            <string>Time2 - DeltaTime</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="buttonD1_plus_DeltaDateDay">
           <property name="text">
            <string>Date1 + DeltaDateDay</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="OutputLayout">
         <item>
          <widget class="QLabel" name="output_Date_label">
           <property name="text">
            <string>OutputDate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="output_Date">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="output_Time_label">
           <property name="text">
            <string>OutputTime</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="output_Time">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
